File: nodejs/7 kyu/50.js
// https://www.codewars.com/kata/55b95c76e08bd5eef100001e/train/javascript

// Passed

function countArara(n) {
  let numberString = "";
  console.log(n);

  if (n % 2 === 0) {
    numberString += "adak ".repeat(Math.floor(n / 2)).trim();
  } else {
    numberString += "adak ".repeat(Math.floor((n - 1) / 2)).trim();
    numberString += " anane";
    numberString = numberString.trim();
  }

  return numberString;
}

const output = countArara(363);
console.log(output);

File: python/7 kyu/50.py
# https://www.codewars.com/kata/564ab935de55a747d7000040/train/python

# Passed

def remove(text, what):
    for char in what:
        for _ in range(what[char]):
            characters = list(text)
            try:
                characters.remove(char)
            except Exception:
                break
                
            text = ''.join(characters)
            
    return text

output = remove('this is a string',{'t':1, 'i':2})
print(output)